packages/design-system/src/components/OcModal/OcModal.vue

```diff
@@ -431,6 +425,12 @@ export default defineComponent({
       cancelButton
     }
   },
+  data() {
```

Comment:
refactor into setup function possible :)?

Reply:
It sure is possible, but this change was a mere linting "fix", moving the data function below the setup function ;)